packages/react-sdk/src/components/Whiteboard/constants.ts

```diff
@@ -14,7 +14,10 @@
  * limitations under the License.
  */
 
-export const whiteboardWidth = 1920;
-export const whiteboardHeight = 1080;
+export const initialWhiteboardWidth = 1980;
+export const initialWhiteboardHeight = 1080;
```

Comment:
```suggestion
export const initialWhiteboardWidth = 1920;
export const initialWhiteboardHeight = 1080;
```

imo, these are the viewport width, which logically we also use initially

```suggestion
export const whiteboardViewportWidth = 1920;
export const whiteboardViewportHeight = 1080;
```